packages/middleware-sdk-transcribe-streaming/src/signer.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { SignatureV4 } from "./signer";
+import { HttpRequest } from "@aws-sdk/protocol-http";
+describe("transcribe streaming", () => {
+  describe("WebSocket request signer", () => {
+    it("should invoke base SigV4 signer correctly", async () => {
+      expect.assertions(4);
+      const mockBaseSigner = {
+        presign: jest.fn()
+      };
+      const signer = new SignatureV4({ signer: mockBaseSigner as any });
+      const toSign = new HttpRequest({
+        headers: {
+          "x-amz-foo": "foo",
+          bar: "bar",
+          "amz-sdk-invocation-id": "123",
+          "amz-sdk-request": "attempt=1",
+          host: "aws.amazon.com"
+        },
+        body: "hello world",
+        query: {
+          prop1: "A",
+          prop2: "B"
+        }
+      });
+      const signed = await signer.sign(toSign);
+      expect(toSign).toMatchObject(signed);
```

Comment:
presign is mocked, so it should return undefined. So I'm not sure how this expect is successful.
Add a mockImplementation for presign function, and check the value returned with signed.

Reply:
It is fixed in 6e86721126c581a82b962d6e4a3ab5c0221dcbe1